installer/conf/omsagent.d/patch_management_immediate_run.conf

```diff
@@ -0,0 +1,12 @@
+<source>
+  type exec
+  tag oms.patch_management_immediate_run
+  command /opt/microsoft/omsconfig/Scripts/PerformInventory.py --InMOF /etc/opt/microsoft/omsagent/conf/omsagent.d/patch_management_inventory.mof --OutXML /etc/opt/omi/conf/omsconfig/configuration/CompletePackageInventory.xml > /dev/null && cat /etc/opt/omi/conf/omsconfig/configuration/CompletePackageInventory.xml
```

Comment:
is it by purpose to let it run when agent restarts? do you need to distinguish the 2 types of data in the service side?

will it cause race condition with the scheduled one, as they use the same output file?